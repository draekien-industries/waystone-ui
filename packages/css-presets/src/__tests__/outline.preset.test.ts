import { outlineCss } from '../outline.presets';

describe('Outline presets', () => {
  it('outlineCss should match snapshot', () => {
    expect(outlineCss).toMatchSnapshot();
  });
});
