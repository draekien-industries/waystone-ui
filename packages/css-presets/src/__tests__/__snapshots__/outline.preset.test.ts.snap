// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Outline presets outlineCss should match snapshot 1`] = `
{
  "active": {
    "outlineColor": [Function],
  },
  "base": {
    "outlineColor": [Function],
    "outlineOffset": 2,
    "outlineStyle": "solid",
    "outlineWidth": 4,
  },
  "focused": {
    "outlineColor": [Function],
  },
}
`;
